cmd/kritis/admission/main.go

```diff
@@ -52,12 +53,24 @@ func main() {
 	flag.BoolVar(&showVersion, "version", false, "kritis-server version")
 	flag.Set("logtostderr", "true")
 	flag.StringVar(&cronInterval, "cron-interval", "1h", "Cron Job time interval as Duration e.g. 1h, 2s")
+	flag.BoolVar(&runCron, "run-cron", false, "Run cron job in foreground.")
 	flag.Parse()
 
 	if showVersion {
 		fmt.Println(version.Commit)
 		os.Exit(0)
 	}
+	// TODO: (tejaldesai) This is getting complicated. Use CLI Library.
+	if runCron {
+		cronConfig, err := getCronConfig()
+		if err != nil {
+			glog.Fatalf("Could not run cron job in foreground: %s", err)
+		}
+		if err := cron.RunInForeground(*cronConfig); err != nil {
+			glog.Fatalf("Error Checking pods: %s", err)
+		}
+		os.Exit(0)
```

Comment:
Use "return" when the exit code is 0. os.Exit can make profilers/debuggers upset, so reserve it for when you need to return a specific non-zero exit code.

Reply:
done!